composer.json

```diff
@@ -18,12 +18,13 @@
         }
     },
     "require": {
+        "php": "^5.6|7.*",
```

Comment:
Why 5.6? It is far far beyond EOL hence not really allowed in any PCI-DSS compliant environment.

Reply:
I see it is still in the travis job matrix.

Reply:
Yes, I think dropping 5.6 and 7.0 will be next, but maybe in a different release.
Probably when adding support for 8.0, as it would even be harder otherwise.

In this change it's just making it clear that 7.4 is also supported, having tests automatically running for it.

Reply:
Up until now, having no upper bound, it would have worked to install the package on PHP 8.0, but wasn't tested sufficiently...
Now it would not work anymore after this is merged.
Maybe it would be better to include in a new minor release also the change mentioned, going with php requirement to `"php": "^7.1|8.0"`, but that would be only after we have php 8 in the pipeline. I would still like to have it as a separate PR.